api/par-api.yaml

```diff
@@ -122,14 +122,17 @@ paths:
         in: query
         description: Used for requesting paged responses, this defines the offset
           of the first Business Rule to return from the start of the list of all matching
-          Business Rules.
+          Business Rules. If not specified this defaults to 0.
         schema:
           type: integer
           default: 0
       - name: limit
         in: query
         description: Used for requesting paged responses, this defines the maximum
-          number of Business Rules to return.
+          number of Business Rules to return. If not specified, all results will be returned.
+          To explicitly request all results, set to a value < 0 (e.g. limit=-1). If set to 0
+          and empty list of results will be returned along with standard details of how many
```

Comment:
Just wording but would "a count of matching results" be better here?

Reply:
made the edit in all places (also fixed another typo that was copy-pasta'd throughout.